delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/OrganizationsRoutesSpec.scala

```diff
@@ -35,18 +33,34 @@ class OrganizationsRoutesSpec extends BaseRouteSpec with IOFromMap {
 
   private val config = OrganizationsConfig(eventLogConfig, pagination, cacheConfig)
 
-  private val aclChecker = AclSimpleCheck().accepted
-  private val aopd       = new OwnerPermissionsScopeInitialization(
+  private val aopd = new OwnerPermissionsScopeInitialization(
     acl => aclChecker.append(acl),
     Set(orgsPermissions.write, orgsPermissions.read)
   )
 
   private lazy val orgs                            = OrganizationsImpl(Set(aopd), config, xas, clock)
   private lazy val orgDeleter: OrganizationDeleter = id => IO.raiseWhen(id == org1.label)(OrganizationNonEmpty(id))
 
-  private val caller = Caller(alice, Set(alice, Anonymous, Authenticated(realm), Group("group", realm)))
-
-  private val identities = IdentitiesDummy(caller)
+  private val superUser                      = User("superUser", Label.unsafe(genString()))
```

Comment:
There is not really a need to generate a realm for each user

Reply:
It's not necessary but it shows that the realm is not relevant to the test